LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/LoRaMessage/LoRaPayloadJoinAccept.cs

```diff
@@ -50,6 +50,10 @@ public class LoRaPayloadJoinAccept : LoRaPayload
         /// </summary>
         public Memory<byte> Fcnt { get; set; }
 
+        public int Rx1DrOffset => (this.DlSettings.Span[0] >> 4) & 0b00000111;

```

Comment:
right shift and then mask it? What is the value in DlSettings.Span[0]? if you right shift it to the first 4 bits, you will always end up with 7 no?

Reply:
I need the three bits located in position 4:6. I could mask it and shift shift, I don't know if the order have an impact here.

Reply:
All good.